catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -536,27 +525,53 @@ public void run() {
 
         public CallableSourceResponse(Source source, QueryRequest request) {
             this.source = source;
-            this.request = request;
+            this.request = new QueryRequestImpl(request.getQuery(), new HashMap<>(request.getProperties()));
         }
 
         @Override
         public SourceResponse call() throws Exception {
-            final SourceResponse sourceResponse =
-                    source.query(new QueryRequestImpl(request.getQuery(), request.getProperties()));
+            SourceResponse sourceResponse = null;
+            try {
+                sourceResponse = source.query(new QueryRequestImpl(request.getQuery(),
+                        request.getProperties()));
+            } catch (UnsupportedQueryException e) {
+                LOGGER.info("Error querying source {}", source.getId(), e);
+            }
+
+            QueryResponse queryResponse = new QueryResponseImpl(request,
+                    sourceResponse == null ? new ArrayList<>() : sourceResponse.getResults(),
+                    true,
+                    sourceResponse == null ? 0L : sourceResponse.getHits(),
+                    sourceResponse == null ? new HashMap<>() : sourceResponse.getProperties());
+
+            for (PostFederatedQueryPlugin service : postQuery) {
```

Comment:
I think this should be done in the `SortedQueryMonitor`.  That will help address Keith's comments about your try-catch in this callable breaking the query monitor logic/error handling.  You should be able to add this to the linked area below and have access to that source's response.

https://github.com/codice/ddf/blob/57b7146277fa791d0567d98beb15999155411882/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java#L142-L148